Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ruby:2.5
```

Comment:
Can this be made to use a version from a `.ruby-version` file?

Reply:
Mike, I think it can be made to match it manually (e.g. 2.3) but that it cannot automatically pull in the version in a `.ruby-version` file.